script.js

```diff
@@ -33,8 +33,8 @@ limitations under the License.
 	}
 
 	var hashRe = /#.*\/[a-f0-9]+$/;
-	var issueRe = /^https:\/\/github.com\/([A-Za-z0-9-]+\/[A-Za-z0-9-]+\/issues\/[0-9]+)/
-	var pullRe = /^https:\/\/github.com\/([A-Za-z0-9-]+\/[A-Za-z0-9-]+\/pull\/[0-9]+)/
+	var issueRe = /^https:\/\/github.com\/([A-Za-z0-9-]+\/[\w-]+\/issues\/[0-9]+)/;
```

Comment:
Maybe I'm being dense, but how does `\w-` match underscores? Did you mean dashes? Doesn't `\w` match spaces? Is that what we want?